packages/labs/cli/src/lib/commands/init.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {Command, CommandOptions} from '../command.js';
+import {run} from '../init/element-starter/index.js';
+import {LitCli} from '../lit-cli.js';
+
+export type Language = 'ts' | 'js';
+export interface InitCommandOptions {
+  lang: Language;
+  name: string;
+}
+
+export const makeInitCommand = (cli: LitCli): Command => {
+  return {
+    kind: 'resolved',
+    name: 'init',
+    description: 'Initialize a Lit project',
+    subcommands: [
+      {
+        kind: 'resolved',
+        name: 'element',
+        description: 'Generate a shareable element starter package',
+        options: [
```

Comment:
Can you add an `out` or `outDir` option? That way if you run this from an npm script or similar (maybe someone wants to do that?) they don't have to `cd` first.